Example/Example/SeriesDetailViewController.swift

```diff
@@ -9,6 +9,92 @@
 import Foundation
 import BothamUI
 
-class SeriesDetailViewController : ExampleViewController {
+private struct Config {
+    static let numberOfColumns = 3
+    static let cellHeight = 174
+    static let headerHeight = 534
+    static let footerHeight = 20
+    static let cellMargin = 20
+}
+class SeriesDetailViewController : ExampleViewController, BothamCollectionViewController, SeriesDetailUI, UICollectionViewDelegateFlowLayout, UICollectionViewDelegate, UICollectionViewDataSource {
+
+
+    var dataSource: BothamCollectionViewDataSource<Comic, ComicCollectionViewCell>!
+    var header: Series?
+
+    @IBOutlet weak var collectionView: UICollectionView!
+
+    override func viewDidLoad() {
+        configureNavigationBar()
+        configureCollectionView()
+        super.viewDidLoad()
+    }
+
+    override func viewWillDisappear(animated: Bool) {
+        navigationController?.navigationBar.setBackgroundImage(nil, forBarMetrics: UIBarMetrics.Default)
+        navigationController?.navigationBar.backgroundColor = UIColor.navigationBarColor
+        navigationController?.navigationBar.shadowImage = nil
+        navigationController?.navigationBar.translucent = false
+    }
```

Comment:
it's missing the super.viewWillDisappear(), it's not important in this case for the functionality, but is good as an example to have it in the code.